server/protocols/webadmin-integration-test/webadmin-integration-test-common/src/test/java/org/apache/james/webadmin/integration/AuthorizedEndpointsTest.java

```diff
@@ -32,24 +30,17 @@
 import org.eclipse.jetty.http.HttpStatus;
 import org.junit.After;
 import org.junit.Before;
-import org.junit.Rule;
 import org.junit.Test;
 
 import io.restassured.RestAssured;
 
-public class AuthorizedEndpointsTest {
-
-    @Rule
-    public DockerCassandraRule cassandra = new DockerCassandraRule();
-
-    @Rule
-    public CassandraRabbitMQAwsS3JmapTestRule jamesTestRule = CassandraRabbitMQAwsS3JmapTestRule.defaultTestRule();
+public abstract class AuthorizedEndpointsTest {
```

Comment:
Also, we should annotate this class with @category(BasicFeature.class) as this is quite critical (and master current behaviour)